src/pages/myGamesPage.js

```diff
@@ -61,10 +62,10 @@ export default function MyGames({
 
     return (
         <>
-            <div className="app-body">
+            <div className="app-body" style={{textAlign:"center"}}>
             <Header />
             <div className="nav_links">
-                <button className="buttonmygames" onClick={() => handleMenuConfirmed()}>
+                <button className="buttonmtgames" onClick={() => handleMenuConfirmed()}>
```

Comment:
Did you mean for this to be buttonmygames?

Reply:
oh god, misspelled issue, sorry about it.

Reply:
Not a problem!

Reply:
For class names let's stick to the word-word format

Reply:
ok, use dash, got it